File: Corredor/Digitales/Java/src/Instrumento.java
public class Instrumento {
    private String nombre;
    private String tipo;
    private double costo;

    public Instrumento(String nombre, String tipo, double costo) {
        this.nombre = nombre;
        this.tipo = tipo;
        this.costo = costo;
    }

    public void mostrar() {
        System.out.println("Instrumento: " + nombre + " | Tipo: " + tipo + " | Costo: $" + costo);
    }
}

File: Corredor/Digitales/JavaScript/Instrumento.js
class Instrumento {
    constructor(nombre, tipo, costo) {
        this.nombre = nombre;
        this.tipo = tipo;
        this.costo = costo;
    }

    mostrar() {
        console.log(`Instrumento: ${this.nombre} | Tipo: ${this.tipo} | Costo: $${this.costo}`);
    }
}

module.exports = Instrumento;